### DISCUSSION TITLE
Question: Any production app published

### DISCUSSION THREAD

**Discussion by hew999** [ethanblake4/flutter_eval#56] [General]
Hi all, I love how this package exist might solve some issue that I faced.
However, I have similar concern with #7  
Just curious about if there anyone using this package in production app published in App store & Google play with no issue?

**Comment by ethanblake4** [comment]
I do not know. However, many of the top apps on the app store use a very similar technology (code-push powered by React Native's "Hermes" JavaScript interpreter) without issue. Though I can't make any promises, it appears that Apple does not have an issue with it as long as you don't hot-update your app to do something that violates their guidelines.

**Comment by hew999** [comment]
> I do not know. However, many of the top apps on the app store use a very similar technology (code-push powered by React Native's "Hermes" JavaScript interpreter) without issue. Though I can't make any promises, it appears that Apple does not have an issue with it as long as you don't hot-update your app to do something that violates their guidelines.


Thank you for replying. I read through  https://github.com/flutter/flutter/issues/14330 
I don't understand how if I use code push to some bug fixes will not violates their policy.
Do u have any scenario to explain about it? 
For example if i involved code push in some widget class to do bug fixes/ code changed, would it be allowed ?

**Comment by ethanblake4** [comment]
Apple's rules are purposefully obtuse and confusing. They can take down your app for almost anything. Will they? Probably not. There is a near-zero% chance your app would be rejected just from *having* flutter_eval in it. I know this isn't a very clear answer, but that's the reality of the iOS App Store.

**Comment by wrbl606** [comment]
An FAQ from a similar project might be helpful here: https://docs.shorebird.dev/faq#does-shorebird-comply-with-play-store-guidelines
